safe/gis/expressions.py

```diff
@@ -39,7 +39,7 @@ def inasafe_impact_analysis_layer(field, feature, parent):
 
     index = layer.fieldNameIndex(field)
     if index < 0:
-        return None
+        return index
```

Comment:
because in our template this function is a parameter for `inasafe_place_value_coefficient()` . If the field is not found in the layer, we want to show `no_data_replacement` string which is `dash` instead of nothing. And somehow if `inasafe_place_value_coefficient()` is being called with None. somehow it doesn't give the result we are expecting